dronegen/aws.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "path/filepath"
+
+// awsRoleSettings contains the information necessary to assume an AWS Role
+//
+// This is intended to be imbedded, please use the kubernetes/mac/windows versions
+// with their corresponding pipelines.
+type awsRoleSettings struct {
+	awsAccessKeyId     value
+	awsSecretAccessKey value
+	role               value
+}
+
+// kuberentesRoleSettings contains the info necessary to assume an AWS role and save the credentials to a volume that later steps can use
```

Comment:
`kuberentesRoleSettings` - typo

Reply:
fixed in https://github.com/gravitational/teleport/pull/17201/commits/c605465c4bcf759a47068d7b64b2ecd3e1532684